### DISCUSSION TITLE
Illustrations selection

### DISCUSSION THREAD

**Discussion by Orion83** [pymedusa/Medusa#10569] [Ideas & Feature requests]
Hello,
is it possible to add the possibility to choose the artworks, posters, banner, etc... from indexer source or locally file?
Some are not the good ones from the automatic selection.

**Comment by p0psicles** [comment]
Yes. Enable metadata in config -> postprocessing. And enable Kodi12 metadata for the images.
Then refresh the show.
Now medusa will have added posters, banners, backgrounds to your show folder.
You can overwite these with your own.

**Comment by Orion83** [comment]
Thx for the tip but was more looking as an option like the Jellyfin's one.
